type_ahead_dropdowns/type_ahead_dropdowns.less

```diff
@@ -0,0 +1,70 @@
+@import '_vars.less';
+
+html {
+  font-size: 10px;
+}
+.type-ahead__search-container {
+  display: flex;
+  .type-ahead__input,
+  .type-ahead__arrow {
+    height: auto;
+    border-radius: .25em;
```

Comment:
`rem` not `em`

(rather than typing it for every instance of `em`, assume i mean it for all of the instances)